project-base/storefront/next-env.d.ts

```diff
@@ -2,4 +2,4 @@
 /// <reference types="next/image-types/global" />
 
 // NOTE: This file should not be edited
-// see https://nextjs.org/docs/pages/building-your-application/configuring/typescript for more information.
+// see https://nextjs.org/docs/basic-features/typescript for more information.
```

Comment:
this change is necessary?

Reply:
I do not know why, but this is showing up all the time, I will revert it